lstchain/io/lstcontainers.py

```diff
@@ -116,6 +116,8 @@ class DL1ParametersContainer(Container):
     ucts_trigger_type = Field(None, "UCTS trigger type")
     trigger_time = Field(None, "trigger time")
 
+    lhfit_call_status(None, "Status of the processing of the event by the LH fit method")
```

Comment:
How this is supposed to work? If `lhfit_call_status` is a field, then it should be declared as other fields above. i.e.
```python
lhfit_call_status = Field(None, "Status of the processing of the event by the LH fit method")
```

Reply:
Thanks for noticing. It is indeed an overview